main.go

```diff
@@ -90,35 +98,35 @@ func (c *Controller) processNextItem() bool {
 func (c *Controller) handleNewCSR(key string) error {
 	obj, exists, err := c.indexer.GetByKey(key)
 	if err != nil {
-		klog.Errorf("Fetching object with key %s from store failed with %v", key, err)
+		glog.Errorf("Fetching object with key %s from store failed with %v", key, err)
```

Comment:
Why are we moving to glog? if there were a reason why in this commit? Can we please use the imperative mood in the commit i.e from "Metrics support" to "Add metrics support"

Reply:
Currently this packahge is usig both glog and klog, https://github.com/openshift/cluster-machine-approver/blob/master/config.go#L7
While debuggin i just made klog logs were not getting printed because klog.Init(nil) is missing so just converted all to glog. Agree that this should be handled in a seperate PR.